bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieImpl.java

```diff
@@ -1082,7 +1082,8 @@ public ByteBuf readEntry(long ledgerId, long entryId)
                 LOG.trace("Reading {}@{}", entryId, ledgerId);
             }
             ByteBuf entry = handle.readEntry(entryId);
-            bookieStats.getReadBytes().add(entry.readableBytes());
+            entrySize = entry.readableBytes();
+            bookieStats.getReadBytes().add(entrySize);
```

Comment:
unnecessary change?

Reply:
it's necessary  for the finally using it

Reply:
I didn't see the `finally` cause refer the entrySize variable. Can you explain more detail? thanks

Reply:
> I didn't see the `finally` cause refer the entrySize variable. Can you explain more detail? thanks

I add the code picture